scripts/backup.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 # PiVPN: Backup Script
 
+CHECK_PKG_INSTALLED='dpkg-query -s'
+
+if [ "$(grep -sEe '^NAME\=' /etc/os-release | sed -Ee "s/NAME\=[\'\"]?([^ ]*).*/\1/")" == 'Alpine' ]
```

Comment:
Simplify:
```suggestion
if grep -qsEe '^NAME\=['\''"]?Alpine('\''|"| |$)' /etc/os-release
```
Is it expected to be `Apline Linux`? If so, check for the whole string instead? Above allows it to end with quotation, space or directly, so should be most flexible, in case if defers across different Alpine Linux versions.

Reply:
Yep, it should be `NAME="Alpine Linux"`